models/dave_robot_models/description/rexrov/model.sdf

```diff
@@ -584,5 +584,22 @@
       <joint_name>thruster8_joint</joint_name>
     </plugin>
 
+    <!-- Sea Pressure Sensor Plugin -->
+    <plugin
+      filename="sea_pressure_sensor"
+      name=" dave_gz_sensor_plugins::SubseaPressureSensorPlugin">
+      <namespace>rexrov</namespace>
```

Comment:
Isn't this suppose to be make msg name as `/rexrov/sensor_output_topic`? Meanwhile, shouldn't `sensor_output_topic` be `pressure`? will we include other sensor msgs into this `sensor_output_topic`?

Reply:
Yes sir, you're correct.